exercises/G-conditionals-2/exercise-3.js

```diff
@@ -18,6 +18,18 @@ let grade2 = 90;
 let grade3 = 70;
 let grade4 = 55;
 
+function calculateGrade(grade){
+  if (grade >= 80) {
+    return "A";
+  } else if (grade > 60 && grade < 80){
+    return "B";
+  } else if (grade >= 50 && grade <= 60){
+    return "C";
+  } else {
+    return "F";
+  }
+}
```

Comment:
Good work - you could remove the second checks in your "else if" sections. For example, in your first "else if" you already know that the grade is less than 80, so you could just check that grade > 60